runbenchmark.py

```diff
@@ -5,6 +5,8 @@
 import shutil
 import sys
 
+import openml
+
```

Comment:
I don't like the idea of adding a dependency to openml in this script and making changes even before knowing if it will be used at all (it is not if loading data from files).
Can't you apply this logic directly in `amlb.datasets.openml` module?